plugins/inputs/cloudwatch/README.md

```diff
@@ -72,8 +72,8 @@ API endpoint. In the following order the plugin will attempt to authenticate.
   ## Configure the TTL for the internal cache of metrics.
   # cache_ttl = "1h"
 
-  ## Metric Statistic Namespace (required)
-  namespace = "AWS/ELB"
+  ## Metric Statistic Namespaces (required)
+  namespaces = ["AWS/ELB"]
```

Comment:
Because namespace is still supported, I recommend keeping it in the sample config.

```suggestion
  ## A single Metric Statistic Namespace (namespace or namespaces is required)
  namespace = "AWS/ELB"
  ## Metric Statistic Namespaces (namespace or namespaces is required)
  namespaces = ["AWS/ELB"]
```

Reply:
@akrantz01 looks like this change was accidentally dropped?

Reply:
That's odd; it might've gotten lost in the rebase. I updated it again.

Reply:
Great thanks!